students/FredBallyns/lesson02/charges_calc.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+'''
+Returns total price paid for individual rentals
+'''
+
+import argparse
+import json
+import datetime
+import math
+import logging
+
+
+def parse_cmd_arguments():
+    try:
+        parser = argparse.ArgumentParser(description='Process some integers.')
+        parser.add_argument('-i', '--input', help='input JSON file', required=True)
+        parser.add_argument('-o', '--output', help='ouput JSON file', required=True)
+        parser.add_argument('-d', '--debug', help='set logging level: 0 default for disabled, 1 errors, 2 for warnings, 3 for debug', required=False, default="0")
+        print("Input Arguments Received")
+        return parser.parse_args()
+    except Exception as exception:
```

Comment:
Be careful when catching generic Exceptions. Typically, you want to be aware of the type of error you want to catch and handle. A DivideByZero error is very different from an IndexError. Your behavior to fix them would be very different, also you don't want to catch errors that you're not expecting.